src/Application/Application.ts

```diff
@@ -139,12 +140,16 @@ export class Application {
   private async populateCoordinatesForIQ() {
     try {
       logMessage('Trying to get sbom from cyclonedx/bom', DEBUG);
-      let files = Lister.getListOfFilesInBasePath(process.cwd());
+      let files = Lister.getListOfFilesInBasePath(join(process.cwd(), 'bin'));
 
-      await Promise.all([this.getHashesFromPath(files), this.muncher.getSbomFromCommand()])
+      console.log(files);
+
+      await Promise.all([this.getHashesFromPath(files, 'bin'), this.muncher.getSbomFromCommand()])
```

Comment:
trivial: Can this just be a return Promise.all

Reply:
I think we need to await this, so we can populate the sbom and stuff for talking to IQ. Do you see another path forward?

Reply:
Ahhh, sorry i misunderstood! I'll mark this convo as resolved.